src/server/knex-connect.js

```diff
@@ -26,6 +27,8 @@ const useSSL = DB_USE_SSL === "1" || DB_USE_SSL.toLowerCase() === "true";
 if (useSSL) pg.defaults.ssl = true;
 // see https://github.com/tgriesser/knex/issues/852
 
+const debug = DB_DEBUG === "1" || DB_DEBUG.toLowerCase() === "true";
```

Comment:
A small nitpick, but I don't see a need for an extra variable here. I think it would be cleaner to make the line down below `config.debug = DB_DEBUG === "1" || DB_DEBUG.toLowerCase() === "true";`

Reply:
Fixed :+1: